spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -1107,6 +1111,10 @@ def test_mpl_backend_change(ipyconsole, qtbot):
 
 @flaky(max_runs=10)
 @pytest.mark.skipif(running_in_ci(), reason="Fails frequently in CI")
+@pytest.mark.skipif(os.name == 'nt',
+                    reason="Failing on Windows. In fact on Windows you can "
+                           "copy with Ctrl + C i.e a KeyboardInterrupt isn't "
+                           "triggered when doing Ctrl + C while debugging")
 def test_ctrl_c_dbg(ipyconsole, qtbot):
```

Comment:
I think we should open a new issue to remove this test because it no longer applies with our new debugger.

Reply:
See https://github.com/spyder-ide/spyder/issues/17498